cluster/mesos/docker/docker-compose.yml

```diff
@@ -29,7 +29,7 @@ mesosmaster1:
 mesosslave:
   hostname: mesosslave
   privileged: true
-  image: mesosphere/mesos-slave-dind:mesos-0.24.0_dind-0.2_docker-1.8.2_ubuntu-14.04.3
+  image: local/kubernetes-mesos-slave-dind:latest
```

Comment:
I don*t like these test images in the docker hub at all. Moreover, they should not even have the mesosphere org because they are not meant to public consumption. I would pretty much prefer to remove them from the hub and use some mechanism here to exclude them from pull before building, see below.